### PR TITLE
Added Hackaday icon #1142

### PR DESCRIPTION
**Pull request by nexus0212**
**Issue: #1142**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color was picked from the website [1] body background.

---
[1] https://hackaday.com/

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1070,6 +1070,11 @@
             "hex": "72EF36",
             "source": "https://www.gumtree.com"
         },
+        {
+            "title": "Hackaday",
+            "hex": "DDDDDD",
+            "source": "https://hackaday.com/"
+        },
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Can you remove the trailing whitespace on this line? It's not much but we prefer not to have it there 🙂 

---

<sup>Linting for this is being implemented in #1349</sup>

File: icons/hackaday.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1.4 3.1 3.1 3.1zM20.9 16.8c-.3 0-.6 0-.9.1l-2.3-2c-.4 1.1-1 2-1.8 2.7l2 1.8v.5c0 1.7 1.4 3.1 3.1 3.1.3 0 .6 0 .9-.1l-2-1.8 2.1-2.4 2 1.8c0-.2.1-.4.1-.6-.1-1.7-1.5-3.1-3.2-3.1zM6.2 14.9l-2.3 2c-.3-.1-.6-.1-.9-.1-1.7 0-3 1.4-3.1 3 0 .2 0 .4.1.6l2-1.8L4.2 21l-2 1.8c.3.1.6.1.9.1 1.7 0 3.1-1.4 3.1-3.1v-.5l2-1.8c-.9-.6-1.6-1.5-2-2.6zM17.5 9.2L20 7c.3.1.6.1.9.1C22.6 7.1 24 5.7 24 4c0-.2 0-.4-.1-.6l-2 1.7-2.1-2.4L21.7 1c-.3-.1-.5-.1-.8-.1-1.7 0-3 1.4-3.1 3v.5l-2.4 2.1c1 .8 1.7 1.7 2.1 2.7z"/><path d="M17.7 12.3C17.7 8.8 15.2 6 12 6s-5.7 2.8-5.7 6.3c0 2.2 1 4.1 2.5 5.2-.1.2-.2.4-.2.6 0 .6.5 1.1 1.1 1.1.6 0 1.1-.5 1.1-1.1V18h.2v.2c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1V18h.2v.2c0 .6.5 1.1 1.1 1.1s1.1-.5 1.1-1.1c0-.2-.1-.4-.2-.6 1.3-1.2 2.3-3.1 2.3-5.3zm-7 .2c-.2.1-.4.3-.6.3-.3.1-.5.2-.8.4l-.3.3c0 .2 0 .4-.1.5-.2.1-.3 0-.4-.1-.3-.2-.5-.5-.6-.8-.1-.3-.1-.6-.1-.8 0-.3.2-.6.4-.9.2-.3.6-.6 1-.6h.3c.3 0 .6.1.9.3.2.2.4.4.6.7.1.2 0 .5-.3.7zm1.8 3.3c-.3.8-.2-.5-.5-.5s-.4 1.2-.6.5c0-.6.3-1.8.6-1.8s.6 1.3.5 1.8zm3-1.9c-.1.1-.3.2-.4.1-.1-.1 0-.3-.1-.5 0-.1-.1-.3-.3-.3-.3-.1-.5-.3-.8-.4l-.6-.3c-.3-.2-.4-.5-.3-.8.1-.3.3-.5.6-.7.3-.2.6-.3.9-.3h.3c.4.1.7.3 1 .6.2.3.4.6.4.9 0 .3 0 .6-.1.8-.1.5-.3.7-.6.9z"/></svg>
```
Comment (ericcornelissen) [inline icons/hackaday.svg]: Looking at the quality of the icon it seems you've optimized a bit too much, could you fix that? 🤔

**Comment by nexus0212** [comment]
I fixed all bugs 💃. @ericcornelissen, I will be grateful to check that everything is okey.

**Comment by ericcornelissen** [comment]
Icon looks good 👌 but it seems the whitespace is still there @nexus0212

**Comment by nexus0212** [comment]
image

> con looks good 👌 but it seems the whitespace is still there @nexus0212

I checked my fragment in `simple-icons.json` file, and I think everything its ok. Could you tell me more where is the problem, because I really don't know whats wrong.

**Comment by nexus0212** [comment]
image

> Icon looks good 👌 but it seems the whitespace is still there @nexus0212

I checked my fragment in `simple-icons.json` file, and I think everything its ok. Could you tell me more where is the problem, because I really don't know whats wrong 🤯

**Comment by ericcornelissen** [comment]
If I have to believe GitHub, it's hiding at the end of the line 😜 

image

**Comment by ericcornelissen** [comment]
Thanks a bunch @nexus0212 🎉

**Pull request merged**